src/App.tsx

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { BrowserRouter as Router, Switch, Route } from 'react-router-dom';
+import './App.css';
+import Home from './components/Home';
+import Charts from './components/Charts';
+import ToggleMenu from './components/ToggleMenu';
```

Comment:
Ideally in big projects its  always good to group your imports. This requires adding an `index.ts` file (note: its NOT `.tsx`) in the `components/` directory root and export all components out from there. And then you can do something like this

```tsx
import { Home, Charts , ToggleMenu } from './components';
```